guides/common/modules/proc_renewing-a-custom-ssl-certificate-proxy.adoc

```diff
@@ -0,0 +1,91 @@
+[id="Renewing_a_Custom_SSL_Certificate_Proxy_{context}"]
+= Renewing a {customssltitle} certificate on {SmartProxyServer}
+
+Use this procedure to renew a {customssl} certificate signed by a Certificate Authority for your {SmartProxyServer}.
+The `{foreman-installer}` command, which the `{certs-generate}` command returns, is unique to each {SmartProxyServer}.
+Do not use the same command on more than one {SmartProxyServer}.
+
+.Prerequisites
+Before renewing a custom server certificate for {SmartProxyServer}, ensure that your {Project} and {SmartProxies} meet the following conditions:
+
+* {ProjectServer} is configured with a custom certificate.
+* {SmartProxyServer} is registered to {ProjectServer}.
+* The {SmartProxyServer} packages are installed.
+
+.Procedure
+You must create a new Certificate Signing Request and send it to the Certificate Authority to sign the certificate.
+In return, you will receive the {SmartProxyServer} certificate and CA bundle.
+You must keep the {SmartProxyServer} SSL certificates on the {ProjectServer} by creating a folder and verifying them through the `katello-certs-check` command.
+
+To renew a {customssl} certificate for your {SmartProxyServer}, complete the following steps:
```

Comment:
```suggestion
```

this IMHO does not add any more value if we have a clear heading and introduction.

Reply:
Addressed :+1: